projects/ngrx.io/content/guide/data/extension-points.md

```diff
@@ -179,6 +179,9 @@ import {
   Pluralizer
 } from '@ngrx/data';
 
+import { Injectable } from '@angular/core';
```

Comment:
Could you move this import to the first line please?

Reply:
Sure thing, that's done.